chain/network/src/peer_manager/connection/mod.rs

```diff
@@ -44,6 +67,11 @@ pub(crate) struct Stats {
 
 /// Contains information relevant to a connected peer.
 pub(crate) struct Connection {
+    // TODO(gprusak): TIER1 connections should minimize the communication:
```

Comment:
I'm surprised that this is a TODO - I thought that this is what we're doing above.

Reply:
true, updated the TODO to actually reflect what is left to be done.